### ISSUE TITLE
Machine Learning Algorithms

### ISSUE THREAD

**Issue by sadn1ck** [codeiiest-dev/Algorithms#3]
###Topic: ML Algos

### Subtopic:

(add in comments)

#### Checks:

- [ ] explanation
- [ ] implementation
- [ ] references
- [ ] extra problems for practice

**Comment by asin1203** [comment]
Assign this to me _please_...

**Comment by Siddhant-K-code** [comment]
Can i add `Linear Regression` ?

**Comment by sadn1ck** [comment]
Yeah sure, feel free to! Have a look through the README and the CONTRIBUTING guide if you're not familiar with mdbook.

**Comment by Siddhant-K-code** [comment]
![image](https://user-images.githubusercontent.com/55068936/100495063-be5e9d00-316d-11eb-9bbb-52f5497c1a27.png)
In this, with which name i should add Machine Learning Algorithms , as: 
 * Machine Learning 
    * Linear Regression

**Comment by sadn1ck** [comment]
[Guide](https://github.com/codeiiest-dev/Algorithms#general)

@Siddhant-K-code create a category for ML under the src folder, then add a markdown file for linear regression and write in markdown.

**Comment by Siddhant-K-code** [comment]
Can it be possible if i add explanation in Markdown file and link Jupyter Notebook Link and add the same under linear regression folder, because for ML Algos. it is not easy to explain without Notebook

**Comment by asin1203** [comment]
Can't we not make a separate point of regression Algorithms and then add
linear regression under it?
Because there are other regression Algorithms as well

**Comment by sadn1ck** [comment]
Sure, feel free to create another issue if you want to separate them out 😄
